packages/gradient-block/src/settings.ts

```diff
@@ -0,0 +1,123 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    Choice,
+    SwitchSize,
+    appendUnit,
+    defineSettings,
+    maximumNumericalOrPixelOrAutoRule,
+    maximumNumericalRule,
+} from '@frontify/guideline-blocks-settings';
+
+import { IconEnum } from '@frontify/fondue';
+import { GradientHeight, GradientOrientation } from './types';
+import { DEFAULT_HEIGHT_VALUE, DEFAULT_ORIENTATION_VALUE } from './constants';
+
+export const IS_CUSTOM_HEIGHT_ID = 'isHeightCustom';
+export const HEIGHT_CUSTOM_ID = 'heightCustom';
+export const HEIGHT_SIMPLE_ID = 'heightSimple';
+
+export const IS_CUSTOM_ORIENTATION_ID = 'isOrientationCustom';
+export const ORIENTATION_CUSTOM_ID = 'orientationCustom';
+export const ORIENTATION_SIMPLE_ID = 'orientationSimple';
+
+export const settings = defineSettings({
+    basics: [
+        {
+            id: 'gradientInspect',
+            type: 'sectionHeading',
+            label: 'Inspect',
+            blocks: [
+                {
+                    id: 'displayCss',
+                    label: 'Display CSS Code',
+                    type: 'switch',
+                    size: SwitchSize.Small,
+                    defaultValue: true,
+                },
+            ],
+        },
+    ],
+    layout: [
+        {
+            id: 'gradientLayout',
+            type: 'sectionHeading',
+            label: '',
+            blocks: [
+                {
+                    id: IS_CUSTOM_HEIGHT_ID,
+                    type: 'switch',
+                    label: 'Height',
+                    switchLabel: 'Custom',
+                    info: 'Determines the block height.',
+                    defaultValue: false,
```

Comment:
You could use the `presetCustomValue` helper from the shared package, to automatically add the preset values for S/M/L to the custom input. See as an example: https://github.com/Frontify/guideline-blocks/blob/8700da7e126c2ee14dfd7224ea15a743c4e2cd2e/packages/divider-block/src/settings.ts#L77

Reply:
oh thats cool, thanks